### PR TITLE
Ev measure updates

### PR DESCRIPTION
**Pull request by amyeallen1**
Hi, per Katherine's advice, I updated the folder structure in this measure so that the resource files are appropriately linked in the measure.xml. This is necessary in order to get the resource files to be included in the BCL. I confirmed that the measure still works as expected. Nick, should you review this, or someone else? This should be a simple review. Thanks! @nllong @kflemin

### CODE REVIEW

**Comment by nllong** [comment]
Can this be merged? Should we also merge this into develop?

**Comment by amyeallen1** [comment]
Yes, this can be merged, and please do go ahead & merge it into develop as well. Thanks, Nick! @nllong

File: lib/measures/add_ev_load/measure.xml [hunk 1]
```diff
@@ -126,6 +128,211 @@
...
       <checksum>2293F4B3</checksum>
     </file>
+    <file>
+      <filename>LICENSE.md</filename>
+      <filetype>md</filetype>
+      <usage_type>license</usage_type>
+      <checksum>E0468DD6</checksum>
+    </file>
+    <file>
+<<<<<<< HEAD
```
Comment (nllong) [inline lib/measures/add_ev_load/measure.xml:138]: @amyeallen1  -- it looks like we have some merge commits in the PR. Do you know what files we are really trying to add? Is it just `add_ev_load` and `add_ems_to_control_ev_charging`?

Comment (amyeallen1) [reply lib/measures/add_ev_load/measure.xml]: Sorry about that! I just made a commit to remove the "head" markup. This is the file that was modified, to add in references to the resource files. @nllong

**Pull request closed without merging**